recipes/doccano/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set version = "1.0.4" %}
+{% set node_version = os.environ.get('NODEJS_VERSION') or '13.6.0' %}
```

Comment:
What is the reason for this very strict pin? Can you put `node 13.*` into `build` and `host` instead?